ratpack-kafka-producer/src/main/java/smartthings/ratpack/kafka/KafkaProducerModule.java

```diff
@@ -25,6 +25,13 @@ protected void configure() {
 		Set<String> servers;
 		String clientId;
 		Long maxBlockMillis = TimeUnit.MINUTES.toMillis(1);
+		boolean enabled = true;
```

Comment:
Whats the proper way to call the driver to get the values? I should do it that way